src/ansys/hps/client/client.py

```diff
@@ -245,6 +282,8 @@ def _auto_refresh_token(self, response, *args, **kwargs):
             response.request.headers.update(
                 {"Authorization": self.session.headers["Authorization"]}
             )
+            if self.dt_client is not None:
+                self.dt_client.binary_config.update(token=self.access_token)
```

Comment:
This is not enough to make sure we capture token expiration issues when making requests to the worker api. We need to be able to inject a refresh hook function into the dt_client. Let's discuss it.

Reply:
Agreed to follow up on this at a later time https://github.com/ansys/pyhps/issues/452